weave

```diff
@@ -48,7 +48,7 @@ weave launch        [--password <pass>] [--trusted-subnets <cidr>,...]
                     [--no-restart] [--resume] [--no-discovery] [--no-dns]
                     [--ipalloc-init <mode>]
                     [--ipalloc-range <cidr> [--ipalloc-default-subnet <cidr>]]
-                    [--plugin]
+                    [--plugin=false --proxy=false]
```

Comment:
This is again saying we can have both or neither, which is wrong.  They are independent.
`[--plugin=false] [--proxy=false]`